src/sentry/static/sentry/app/views/organizationEventsV2/modalPagination.tsx

```diff
@@ -88,7 +104,9 @@ ModalPagination.propTypes = {
   event: SentryTypes.Event.isRequired,
 };
 
-const StyledLink = styled(Link, {shouldForwardProp: isPropValid})`
+const StyledLink = styled(Link, {
+  shouldForwardProp: isPropValid,
+})<{theme: any; disabled: boolean; fontSizeMedium: string; isLast: boolean}>`
   color: ${p => (p.disabled ? p.theme.disabled : p.theme.gray3)};
   font-size: ${p => p.fontSizeMedium};
```

Comment:
This might need to be`p.theme.fontSizeMedium` I don't think we have a prop named fontSizeMedium.